packages/ubuntu_bootstrap/lib/pages/storage/manual/manual_storage_model.dart

```diff
@@ -101,8 +102,7 @@ class ManualStorageModel extends SafeChangeNotifier {
   }
 
   /// Notifies selection changes for auto-scrolling.
-  Stream get onSelectionChanged => _selectionController.stream;
-  final _selectionController = StreamController.broadcast();
+  ChangeNotifier selectionChangedNotifier = ChangeNotifier();
```

Comment:
Nice, that's much simpler!

Reply:
Yeah, not a big fan of emitting nulls over streams :sweat_smile: